docker-compose/kafka-integration-test/v2/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+services:
```

Comment:
let's get rid of `kafka-integration-test/` folder and keep everything under `kafka/`